ui.go

```diff
@@ -1171,6 +1171,15 @@ func (ui *ui) readNormalEvent(ev tcell.Event, nav *nav) expr {
 				ui.menuBuf = nil
 				return draw
 			}
+			if tev.Modifiers() == tcell.ModCtrl && !strings.HasPrefix(val, "<c-") {
+				val = val[:1] + "c-" + val[1:]
+			}
+			if tev.Modifiers() == tcell.ModShift {
```

Comment:
You could support more bindings by checking `tev.Modifiers() & tcell.ModShift`, but that would increase the chance for bugs (you might end up with `<s-s-space>` or something like that).

Reply:
I have thought about supporting multiple modifiers at the same time, like Ctrl + Shift, but at this stage I think such mappings are rare and not worth the extra complexity.